modules/end-user-guide/pages/supported-languages.adoc

```diff
@@ -1,10 +1,18 @@
 :_content-type: REFERENCE
-:description: Supported languages
+:description: Stack samples
 :keywords: user-guide, languages
-:navtitle: Supported languages
+:navtitle: Stack samples
 :page-aliases:
 
 [id="supported-languages"]
-= Supported languages
+= Stack samples
 
```

Comment:
Here's my suggested text for a rewrite of the NOTE from the bottom of the file.
IMO, this is a better place for this bit, and the reader better read it before reading the list of stacks.

```suggestion
TIP: Do not confuse stack samples and programming languages that are supported by the in-browser IDE that you select for a workspace. For programming language support, consult the documentation of the IDE and its applicable extensions.

```

Reply:
@l0rd, does the above suggestion address your concern here?